UIKitDZ/UIKitDZ/Modules/OrderModule/Controller/OrderViewController.swift

```diff
@@ -0,0 +1,137 @@
+// OrderViewController.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// Order details screen
+class OrderViewController: UIViewController {
+    // MARK: - Private Properties
+
+    private lazy var dismissButton: UIButton = {
+        let button = UIButton()
+        button.setImage(UIImage(named: "dismissButton"), for: .normal)
```

Comment:
```suggestion
        button.setImage(UIImage(named: "dismissButton"), for: .normal)
```
"dismissButton" в константы